addons/spreadsheet/static/src/list/list_actions.js

```diff
@@ -9,7 +9,7 @@ export const SEE_RECORD_LIST = async (position, env) => {
     if (!cell) {
         return;
     }
-    const { args } = getFirstListFunction(cell.content);
+    const { args } = getFirstListFunction(cell.compiledFormula.tokens);
```

Comment:
I think tokens are never changed when adding/removing rows or columns. And annoyingly references.
With a formula `=ODOO.LIST(1, 1, A1)`, and then insert a column before A, I think the reference token would still be A1 instead of B1, leading to a wrong value when evaluating it with `evaluateFormula`